### PR TITLE
allow setting annotations on node(pool)

### PR DESCRIPTION
**Pull request by auhlig**

### CODE REVIEW

File: pkg/templates/ignition.go [hunk 1]
```diff
@@ -98,6 +101,9 @@ func (i *ignition) GenerateNode(kluster *kubernikusv1.Kluster, pool *models.Node
 		for _, userLabel := range pool.Labels {
 			nodeLabels = append(nodeLabels, userLabel)
 		}
+		for _, userAnnotation := range pool.Annotations {
+			nodeAnnotations = append(nodeAnnotations, userAnnotation)
```
Comment (databus23) [inline pkg/templates/ignition.go]: I don't understand where this is used. This is a new variable and it I'm missing how this is passed to ignition.

**Comment by databus23** [comment]
Can you elaborate why this is needed a bit? We already label each node pool with something. it seems to me this just adds another form of tag to a node pool. Couldn't you also use the nod pool label for selecting what do do?

**Comment by auhlig** [comment]
Want to control k8s operators on node level, like `give-me-a-fip: "true"` annotation the operator would do so.
Would also work with `labels` of course. I though `annotations` would make since the above is non-identifying to node.

**Comment by auhlig** [comment]
scratch that. not possible to register node with annotations 😓 just using labels now

**Pull request closed without merging**